virtual-env.ipynb

```diff
@@ -26,78 +31,116 @@
     "Virtual environments make it easy to cleanly separate projects and avoid problems with different dependencies and version requirements across components.\n",
-    "The conda command-line interface (CLI) is the preferred interface for managing installations and virtual environments with the Anaconda Python distribution.\n",
-    "The Jupyter Notebook service of the Iguazio Data Science Platform (\"the platform\") comes pre-deployed with conda.\n",
-    "This tutorial explains how to use conda to create a Python virtual environment that will be available as a custom kernel in Jupyter Notebook.\n",
-    "For general information about using conda to create virtual environments, see the [conda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)."
+    "The Conda command-line interface (CLI) is the preferred interface for managing installations and virtual environments with the Anaconda Python distribution.\n",
+    "The Jupyter service of the Iguazio Data Science Platform (\"the platform\") comes pre-deployed with Conda.\n",
+    "This tutorial explains how to use Conda to create a Python virtual environment that will be available as a custom kernel in JupyterLab.\n",
+    "For general information about using Conda to create virtual environments, see the [Conda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)."
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"preconfigured-conda-environments\"></a>\n",
+    "## Preconfigured Conda environments\n",
+    "\n",
+    "Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod:\n",
+    "- **base**: This is the default base environment that includes the Conda binaries. It should not be used for development purposes.\n",
+    "- **jupyter**: This environment incudes the JupyterLab server and all of its dependencies. It should not be used for development purposes.\n",
+    "- **mlrun-base**: This environment includes MLRun and all its dependencies preinstalled.\n",
+    "- **mlrun-extended**: This environment includes all the packages from mlrun-base, as well as additional packages such as TensorFlow, PyTorch, and scikit-learn, which are required for the demo notebooks.\n",
+    "\n",
+    "While Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod, it's important to note that any Conda installations to these environments are not persistent. They will be reset when the Jupyter service is restarted.\n",
+    "\n",
+    "However, when these Conda environments are activated, the `PIP_PREFIX` and `PYTHONPATH` environment variables are automatically set to the data mount directory of the running user: `/User/.pythonlibs/<environment name>`. This means that PIP installations will persist even after restarting the Jupyter pod.\n",
```

Comment:
"However, when these Conda environments are activated, the `PIP_PREFIX` and `PYTHONPATH` environment variables are automatically set to the data mount directory of the running user: `/User/.pythonlibs/<environment name>`. This means that PIP installations persist even after restarting the Jupyter pod.\n",